### DISCUSSION TITLE
monai-weekly

### DISCUSSION THREAD

**Discussion by thibaultdvx** [Project-MONAI/MONAI#8052] [General]
Hi, just two quick questions on `monai-weekly`: Does it correspond to the latest commit of the week on your branch `dev`? How stable is it?

Thanks for your great work!

**Comment by KumoLiu** [comment]
Yes, monai-weekly generally corresponds to the latest commit of the week (released on every Sunday, detailed release notes could be found [here](https://pypi.org/project/monai-weekly/)). Since it comes from `dev` branch, monai-weekly may not be as stable as the main or release versions, it is better for development or research purposes rather than production environment.
Thanks.

**Reply by thibaultdvx** [reply]
Very clear, thanks a lot!

**Comment by thibaultdvx** [comment]
Hi,
Another question came to my mind, regarding your releasing policy.
From what I can see, if you follow semantic versioning rules (_major.minor.patch_), there is a minor release every six months and some occasional patches. However, some of your patches seem to be more than that (e.g. [1.3.1](https://github.com/Project-MONAI/MONAI/releases/tag/1.3.1)). So, could you please provide some clarifications on your releasing policy?

Best,
